### ISSUE TITLE
Request for APUNOn

### ISSUE THREAD

**Issue by mzakiab** [aprsorg/aprs-deviceid#184]
```
tocall: APUNOn
vendor: 9W2KEY
model: APRSUNO
class: tracker
os: embedded
contact: mzakiab@gmail.com
```

**Comment by hessu** [comment]
Hello,

You already have an allocation for APKEYn, where only 1 device is using that allocation. It is not an effective use of the device index if each user gets an allocation for each of their own devices, the index will grow too fast and become hard to maintain. The index is intended for devices and software which are used by many people, and to identify the software, not the user.

Are you planning to distribute this software?

**Comment by mzakiab** [comment]
yes, I plan to distribute this software, and in the future will develop more software and other hardware. Thanks

**Comment by hessu** [comment]
Hi! Please use either the existing APKEY allocation (which allows numeric versions as the last character), or an experimental tocall in the APZ prefix (APZUNO for example) until the software is ready for distribution. I prefer not to allocate a lot of different tocalls to a single user for software which is not distributed, the database would grow a bit too much if we did that.
